change/@ni-nimble-components-df093e6a-7cba-46ed-8d5d-07670af90719.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It'd be good to mention in the date column docs that it respects the lang token to do locale-specific formatting (and maybe link to the new theme provider docs). I think adding it to `dateTextColumnDescription` is fine, even though I believe that's not yet visible to Storybook users since we don't have a Docs page for the columns yet. We will someday!